railties/lib/rails/rack/server.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+# :enddoc:
+
+module Rails
+  module Rack
```

Comment:
I think you should consider renaming this module to `Rackup`, two reasons:

1. Anyone who writes `Rack::` within the Rails namespace will get this module instead of `::Rack`.
2. Going forward, `::Rackup::Server` is the proper name.

Reply:
Addressed (2) in [05e9464](https://github.com/rails/rails/pull/47885/commits/05e946410bacc89158007cbd641a58752b7edc23).

For (1) the same will be true for `Rackup::` inside the Rails namespace. There only thing we could do is introduce an entirely new namespace for the rackup server constant. Personally I prefer the current approach, but happy to adjust if anyone feels strongly about the name.

Reply:
Also my reasoning for the original name is because there is already a `rails/rack/logger`, so putting our Rack stuff near together made sense.
https://github.com/rails/rails/blob/main/railties/lib/rails/rack/logger.rb